pkg/skaffold/errors/buildProblems.go

```diff
@@ -23,6 +23,11 @@ import (
 
 const (
 	PushImageErrPrefix = "could not push image"
+	// Error Prefix matches error thrown by Docker
+	// See https://github.com/moby/moby/blob/master/client/errors.go#L18
+	DockerConnectionFailedPrefix = ".*Cannot connect to the Docker daemon.* "
```

Comment:
`...Prefix` seems redundant on every variable and also isn't accurate since the pattern will match any substring and not just prefix.

If we want to group these under a common name how about:

```go
var errorPatterns = struct {
	PushImageErr string
}{
	PushImageErr: "could not push image",
}
```
which although verbose looks better when referenced:
```go
regexp:  re(fmt.Sprintf(".*%s.* denied: .*", errorPatterns.PushImageErr)),
```

Reply:
sounds good.